docs/canonicalk8s/snap/reference/upgrading.md

```diff
@@ -0,0 +1,51 @@
+# Upgrade notes
+
+## Upgrade Instructions
```

Comment:
I think this is redundant. We should remove it and make all the subsequent headings one less